applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -455,6 +459,9 @@ compose_voicemail(#mailbox{keys=#keys{login=Login
                         {'ok', Flow} -> {'branch', Flow};
                         {'error', _R} -> record_voicemail(tmp_file(Ext), Box, Call)
                     end;
+                Continue ->
+                    lager:info("caller chose to continue to the next element in the callflow"),
+                    'ok';
```

Comment:
I was aware of this, however decided to be explicit. Have changed according to your suggestion.